dss_client/include/dss_client.hpp

```diff
@@ -42,7 +42,7 @@
 namespace dss {
 	namespace py = pybind11;
 
-#define DSS_VER    "20210217"
+#define DSS_VER    "71d317f"
```

Comment:
Dont check in this file. Do `git checkout dss_client/include/dss_client.hpp` before doing git commit